modules/telco-core-cpu-partitioning-and-performance-tuning.adoc

```diff
@@ -7,7 +7,10 @@
 = CPU partitioning and performance tuning
 
 New in this release::
-* No reference design updates in this release.
+* Disable RPS - resource use for pod networking should be accounted for on application CPUs.
+* Better isolation of control plane on schedulable master nodes
+* Support for schedulable control-plane in NUMAResourcesOperator
```

Comment:
Use either 
- the NUMA Resources Operator
- the `NUMAResourcesOperator` resource